src/constants/layer.js

```diff
@@ -0,0 +1,4 @@
+export const layerVisibility = Object.freeze({
+    VISIBLE: 'visible',
```

Comment:
In my honest opinion this is less readable than putting `'visible'`

Reply:
In this case is not a matter of readability, but of reusability.